### PR TITLE
fix: force exit when runner complete

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

mocha4 changed the default behavior when runner complete

### CODE REVIEW

File: test/fixtures/no-exit/test/foo.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const http = require('http');
+const assert = require('assert');
+
+describe('mocha-test', () => {
+  it('should work', () => {
+    assert(true);
+  });
+});
+
+const server = http.createServer((req, res) => {
```
Comment (atian25) [inline test/fixtures/no-exit/test/foo.test.js]: 要放到 it 里面吧？

Comment (popomore) [reply test/fixtures/no-exit/test/foo.test.js]: 没关系，只要能不退出就可以

Comment (atian25) [reply test/fixtures/no-exit/test/foo.test.js]: 写到 `describe` 里面的 before 好点吧，mocha 的非 it 内容，好像会影响到 `only` / `skip` 这些的。

Comment (popomore) [reply test/fixtures/no-exit/test/foo.test.js]: 这个是测试的 fixtures

**Comment by atian25** [comment]
@popomore 你发吧

**Comment by popomore** [comment]
+ egg-bin@4.3.5

**Pull request merged**